Uplift/Views/Cells/ActivityCollectionViewCell.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  ActivityTableViewCell.swift
+//  Uplift
+//
+//  Created by Elvis Marcelo on 4/26/23.
+//  Copyright © 2023 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class ActivityCollectionViewCell: UICollectionViewCell {
+
+    let iconImageView = UIImageView()
+    let iconLabel = UILabel()
+    let starButton = UIButton()
+    var activity: Activity!
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setupViews()
+        setupConstraints()
+    }
+
+    func configure(for activity: Activity) {
```

Comment:
**Almost all of the time, your class functions and attributes should be private**

Reply:
configure() needs to be public to be accessed in didSelectAt by the cell